logic/dapp.js

```diff
@@ -186,12 +186,14 @@ DApp.prototype.verify = function(transaction, sender, cb, tx) {
 		}
 	}
 
-	return (tx || library.db).dapps
-		.getExisting({
+	return library.storage.entities.Transaction.getExistingDapps(
+		{
 			name: transaction.asset.dapp.name,
 			link: transaction.asset.dapp.link || null,
-			transactionId: transaction.id,
-		})
+			transactionId_ne: transaction.id,
+		},
```

Comment:
You don't need this method it can be replaced by: 

```
library.storage.entities.Transaction.isPersisted([{name: ransaction.asset.dapp.name, transactionId_ne: transactio.id}, {link: ransaction.asset.dapp.link, transactionId_ne: transactio.id}])
```

Reply:
It uses a complex where query that storage only supports by building the filter by repeating filters. I would say for the sake of getting it merged to keep it as is and once we iterate over the filters for storage we remove this method. What you think?

Reply:
It's using the returned values in the logic I don't think it's a direct replacement with isPersisted that will return true|false

Reply:
Changed to `get()` with filters as the rows being used for calling cb with different errors. We could refactor the whole method in the future but I think that part would be extending the scope of this PR.